### ISSUE TITLE
Class 'Gettext\Translations' not found in phar

### ISSUE THREAD

**Issue by benoitchantre** [wp-cli/i18n-command#81]
I'm trying this command and got the following fatal error:

```
Fatal error: Uncaught Error: Class 'Gettext\Translations' not found in phar:///usr/local/bin/wp/vendor/wp-cli/i18n-command/src/MakePotCommand.php:340
Stack trace:
#0 phar:///usr/local/bin/wp/vendor/wp-cli/i18n-command/src/MakePotCommand.php(215): WP_CLI\I18n\MakePotCommand->handle_arguments(Array, Array)
#1 [internal function]: WP_CLI\I18n\MakePotCommand->__invoke(Array, Array)
#2 phar:///usr/local/bin/wp/vendor/wp-cli/wp-cli/php/WP_CLI/Dispatcher/CommandFactory.php(89): call_user_func(Array, Array, Array)
#3 [internal function]: WP_CLI\Dispatcher\CommandFactory::WP_CLI\Dispatcher\{closure}(Array, Array)
#4 phar:///usr/local/bin/wp/vendor/wp-cli/wp-cli/php/WP_CLI/Dispatcher/Subcommand.php(425): call_user_func(Object(Closure), Array, Array)
#5 phar:///usr/local/bin/wp/vendor/wp-cli/wp-cli/php/WP_CLI/Runner.php(365): WP_CLI\Dispatcher\Subcommand->invoke(Array, Array, Array)
#6 phar:///usr/local/bin/wp/vendor/wp-cli/wp-cli/php/WP_CLI/Runner.php(388): WP_CLI\Runner->run_command(Array, Array)
#7 phar:///usr/local/bin/wp/vendor in phar:///usr/local/bin/wp/vendor/wp-cli/i18n-command/src/MakePotCommand.php on line 340

```

WP-CLI 2.0.0 (phar).
OS: OS X 10.13

**Comment by schlessera** [comment]
@benoitchantre This is a known bug in the current stable Phar. It will be fixed in the v2.0.1 Phar I am releasing today.

In the mean-time, you can use the latest nightly build to use the `i18n` commands:
`wp cli update --nightly`

**Comment by benoitchantre** [comment]
Thank you for the info (and all your work).
